image/l0_smoothing/example_usage.py

```diff
@@ -0,0 +1,35 @@
+from tensorflow.keras.preprocessing import image
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.